transcom/mymove
# INT B-20249 incorrect gbloc prime api

## [B-20249](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20249)

## Summary

This is to fix `originDutyLocationGBLOC` under orders object in Prime Api getMoveTaskOrder.

Currently for Prime api, `originDutyLocationGBLOC` is displaying GBLOC for SC(duty location) and this needs to be modified to TOO's GBLOC which is determined by 1st shipment's pickup zip.

### How to test

1. Create a move with all the duty locations set to Scott AFB.
2. Create a HHG shipment and use `32223` as pickup location zip.
3. Finish creating and submit.
4. As a SC -> Check to see if the Origin GBLOC in the header show `KKFA` and Submit/Counsel the move.
5. As a TOO -> find the move and open it and make sure you see `CNNQ` as Origin GBLOC in the header.
6. As a Prime -> Find the move and open it and open dev tools and make sure `originDutyLocationGBLOC` say CNNQ.

## Screenshots
![image](https://github.com/transcom/mymove/assets/146856854/024e714f-396d-40c5-92ea-91c9afea6b22)

![image](https://github.com/transcom/mymove/assets/146856854/076ad140-0e06-4c1a-b3ef-791664c23f06)
